Gamma/Strategy.h

```diff
@@ -693,10 +693,13 @@ namespace phsInc{
 		/// \param[in] pulseWidth	pulse width in cycles
 		/// \param[in] length		total pulse length in cycles
 		Rhythm& pulse(uint8_t pulseWidth, uint8_t length){
+#pragma warning(push)
```

Comment:
All looks good.

I do not understand these pragmas. Are these MSVC specific?

Reply:
Yes, MSVC pragma warning allows you to silence specific warning numbers in a scoped manner. In the one that you highlighted, MSVC tosses a warning about assigning a negative value to an unsigned variable. I put the pragmas up in situations where I didn't want to potentially break behavior between compilers.

One major warning that I didn't close (due to more invasive changes) is a warning about an unnamed union and struct in Polar (Types.h). In a typical compilation, this tosses 500+ warnings about it being a non-standard compiler extension. I haven't run into trouble compiling it with a number of toolchains, but it might be worth modifying.

Reply:
Okay, thanks. I'll fold this in. I should be able to fix the first warning in a more portable fashion via 

    auto ones = std::numeric_limits<uint64_t>::max();

I will look into the unnamed union and struct. I think these are part of the C11 standard, but not C++. It might be tricky finding a workaround.